tests/bulk_create/tests.py

```diff
@@ -633,8 +633,18 @@ def test_update_conflicts_unique_fields_pk(self):
         "supports_update_conflicts", "supports_update_conflicts_with_target"
     )
     def test_update_conflicts_two_fields_unique_fields_both(self):
-        with self.assertRaises((OperationalError, ProgrammingError)):
+        try:
```

Comment:
I checked the usage of `supports_update_conflicts_with_target`, and it turns out that targed refers to the fields that are used to determine the presence of a conflict, not the fields that should be updated. SingleStore does not support this setting, so we should set `supports_update_conflicts_with_target` to False and re-run the tests.